core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -6,14 +6,28 @@ echo '{{.EtcdIndexEnvVarName}}={{extra.etcdIndex}}' >> {{.EtcdNodeEnvFileName}}
 export COREOS_PRIVATE_IPV4 COREOS_PRIVATE_IPV6 COREOS_PUBLIC_IPV4 COREOS_PUBLIC_IPV6
 REGION=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document | jq -r '.region')
 USERDATA_FILE=userdata-etcd
-while ! /usr/bin/rkt run \
-   --net=host \
-   --volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf  \
-   --volume=awsenv,kind=host,source=/var/run/coreos,readOnly=false --mount volume=awsenv,target=/var/run/coreos \
-   --trust-keys-from-https \
-   {{.AWSCliImage.Options}}{{.AWSCliImage.RktRepo}} --exec=aws -- s3 --region $REGION  cp {{ $S3URI }}  /var/run/coreos/$USERDATA_FILE; do
-  sleep 1
-done
+
+aws() {
+  while ! /usr/bin/rkt run \
+     --net=host \
+     --volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf  \
+     --volume=awsenv,kind=host,source=/var/run/coreos,readOnly=false --mount volume=awsenv,target=/var/run/coreos \
+     --trust-keys-from-https \
+     {{.AWSCliImage.Options}}{{.AWSCliImage.RktRepo}} --exec=aws -- "$@"; do
+    sleep 1
+  done
+}
+aws s3 --region $REGION  cp {{ $S3URI }}  /var/run/coreos/$USERDATA_FILE
+
+INSTANCE_ID=$(curl -s http://169.254.169.254/latest/meta-data/instance-id)
+STACK_NAME=$(aws ec2 describe-tags --region "$REGION" --filters \
+  "Name=resource-id,Values=$INSTANCE_ID" \
+  "Name=key,Values=aws:cloudformation:stack-name" \
+  --output json \
+| jq -r ".Tags[].Value")
```

Comment:
@redbaron Now jq seems to be complaining like:
```
jq: error (at <stdin>:1): Cannot index array with string "Tags"
parse error: Invalid numeric literal at line 1, column 22
```

I wondered why it happened at first glance but running the aws command alone revealed the issue:

```
# /usr/bin/rkt run --net=host --volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf --volume=awsenv,kind=host,source=/var/run/coreos,readOnly=false --mount volume=awsenv,target=/var/run/coreos --trust-keys-from-https quay.io/coreos/awscli:master --exec=aws -- ec2 describe-tags --region ap-northeast-1 --filters Name=resource-id,Values=i-0e4224cb0ffdb4ce7 Name=key,Values=aws:cloudformation:stack-name --output json
[ 1506.407457] awscli[5]: {
[ 1506.408016] awscli[5]:     "Tags": [
[ 1506.408338] awscli[5]:         {
[ 1506.408610] awscli[5]:             "ResourceType": "instance",
[ 1506.408878] awscli[5]:             "ResourceId": "i-0e4224cb0ffdb4ce7",
[ 1506.409197] awscli[5]:             "Value": "pr682a-Controlplane-11TWK5UX7W875",
[ 1506.409551] awscli[5]:             "Key": "aws:cloudformation:stack-name"
[ 1506.409863] awscli[5]:         }
[ 1506.410205] awscli[5]:     ]
[ 1506.410519] awscli[5]: }
```

Adding the `--interactive` flag to `rkt run` seems to resolve the issue:

```
/usr/bin/rkt run --interactive --net=host --volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf --volume=awsenv,kind=host,source=/var/run/coreos,readOnly=false --mount volume=awsenv,target=/var/run/coreos --trust-keys-from-https quay.io/coreos/awscli:master --exec=aws -- ec2 describe-tags --region ap-northeast-1 --filters Name=resource-id,Values=i-0e4224cb0ffdb4ce7 Name=key,Values=aws:cloudformation:stack-name --output json
{
    "Tags": [
        {
            "ResourceType": "instance",
            "ResourceId": "i-0e4224cb0ffdb4ce7",
            "Value": "pr682a-Controlplane-11TWK5UX7W875",
            "Key": "aws:cloudformation:stack-name"
        }
    ]
}
```